altair/utils/save.py

```diff
@@ -160,12 +158,9 @@ def perform_save():
                 **kwargs,
             )
             write_file_or_filename(
-                fp,
-                mimebundle["text/html"],
-                mode="w",
-                encoding=kwargs.get("encoding", "utf-8"),
+                fp, mimebundle["text/html"], mode="w", encoding=encoding
             )
-        elif format in ["png", "svg", "pdf", "vega"]:
+        elif format in {"png", "svg", "pdf", "vega"}:
```

Comment:
I've only made the change here, but this lint is from [ruff PLR6201](https://docs.astral.sh/ruff/rules/literal-membership/), if you wanted to add to `pyproject.toml` for the quick fix everywhere.

Reply:
Let's keep the original list here. It's quite a common codestyle throughout the codebase and we almost never use sets. We could of course change but I'd prefer consistency for now.